alerting/claranet-ingest/versions.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     datadog = {
       source  = "DataDog/datadog"
-      version = "3.37.0"
+      version = ">= 3.0.0"
```

```suggestion
      version = "~> 3.0"
```
To avoid bumping at 4.0 without warning.